charts/aws-ebs-csi-driver/templates/controller.yaml

```diff
@@ -6,6 +6,10 @@ metadata:
   namespace: {{ .Release.Namespace }}
   labels:
     {{- include "aws-ebs-csi-driver.labels" . | nindent 4 }}
+  {{- with .Values.controller.deploymentAnnotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
```

Comment:
Indentation is inconsistent with the rest of this file, suggest:
```
  annotations:
    {{- with .Values.node. deploymentAnnotations }}
    {{- toYaml . | nindent 4 }}
    {{- end }}
```

Reply:
Counterpoint: https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/489be46f0252248e8534f636e049fff98ac380ca/charts/aws-ebs-csi-driver/templates/controller.yaml#L31-L33

We can have a followup PR that makes this consistent across all templates.